ptftplib/tftpclient.py

```diff
@@ -19,6 +20,14 @@
 # along with pTFTPd.  If not, see <http://www.gnu.org/licenses/>.
 
 
+from __future__ import print_function
+
+try:
+    # noinspection PyShadowingBuiltins
+    input = raw_input  # Py2
```

Comment:
I like `six` for big libraries but since the only really compatibility thing we are looking at is `input` vs `raw_input` it is easy to just overwrite this one. You avoid a dependency as well.  :)